prisma/schema.prisma

```diff
@@ -2,6 +2,7 @@
 
 generator client_py {
   provider = "prisma-client-py"
+  interface = "sync"
```

Comment:
what's this do?

Reply:
- this generates a synchronous prisma client, 
- the default is an async client, which needs to be awaited 
- the async client was hard to configure with the logger, so I went for the simple solution